### ISSUE TITLE
is CRF available?

### ISSUE THREAD

**Issue by hbk16** [zhongzisha/ISBI2018_PETCT_Segmentation#1]
Having looked into your code, I found that Conditional Random Field is also contained, but was not mentioned in your ISBI paper. And the function crfrnn3d is absent. Does CRF work? How is it implemented?

**Comment by hbk16** [comment]
Having looked into your code, I found that Conditional Random Field is also contained, but was not mentioned in your ISBI paper. And the function crfrnn3d is absent. Does CRF work?

**Comment by zhongzisha** [comment]
Yes, it worked, but in our experiments, we didn't get significant improvement from that. The crfrnn3d was implemented by following the https://github.com/sadeepj/crfasrnn_keras. Basically modify the files to accept the 3D inputs.
